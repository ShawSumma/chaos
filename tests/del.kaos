bool var1 = true
num var2 = 5
str var3 = 'string1'
number var4 = 45.6
del var2
symbol_table
