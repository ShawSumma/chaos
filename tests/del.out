[start] =>
	{id: 0, name: var1, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 0, 2nd_type: 0, value_type: 0, role: 0, param_of: } =>
	{id: 2, name: var3, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{id: 3, name: var4, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 1, 2nd_type: 0, value_type: 2, role: 0, param_of: } =>
[end]
